resources/builder.inc.sh

```diff
@@ -631,8 +663,8 @@ builder_describe() {
       else
         # If the target name matches a folder name, implicitly
         # make it available as a child project
-        if [[ -d "$THIS_SCRIPT_PATH/${value:1}" ]]; then
-          target_path="${value:1}"
+        if [[ -d "$THIS_SCRIPT_PATH/$_builder_child_base/${value:1}" ]]; then
+          target_path="$_builder_child_base/${value:1}"
```

Comment:
I'm fine with that for the builder variables since we consider the `:` to be part of a target's name and `--` part of an option's name, but that sits... less well with me for paths.  Having that intermediate `/` makes it much more obvious at a glance that the constructed string _is_ a path, after all.  (Also, I think I've propagated the `/`-including pattern significantly within Web's scripts.)

That said, I can make the change if you're sure.

Reply:
I'm sure, because I don't want paths to be `foo//bar` which is what we'll get if `$_builder_child_base` is `""`.

Reply:
Also, I _might_ be wrong, but "repeated slashes" still parse as a proper path, right?  I'd rather see a double-slash than a no-slash that causes sporadic file-referencing errors.

Reply:
You're missing the other change I made. https://github.com/keymanapp/keyman/pull/8290#discussion_r1117863282